app/main.py

```diff
@@ -30,3 +33,8 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/eventedit")
```

Comment:
Maybe `/event/edit`?

Reply:
not @app.put("/event")?

Reply:
@Gonzom this is specific for the get method, not post or put

Reply:
Ah, the get request is to reach the edit page, not to edit the event. My mistake!

Reply:
@yammesicka 👍